### PR TITLE
[native] Set PRESTO_ENABLE_PARQUET in test-native CI job

### PR DESCRIPTION
**Pull request by isadikov**
This PR add `-DPRESTO_ENABLE_PARQUET=ON` to test-native CI job to include Parquet as part of Prestissimo. This will allow us to run Prestissimo tests with Parquet in CI.

```
== NO RELEASE NOTE ==
```

### CODE REVIEW

**Comment by isadikov** [comment]
@aditi-pandit Can you review this PR? Thanks.

**Comment by isadikov** [comment]
@aditi-pandit @majetideepak Could you review this PR? This is needed to enable Parquet in test-native CI builds. Thanks.

**Review by majetideepak — APPROVED** [review]

**Comment by majetideepak** [comment]
@isadikov I do not have permission to merge this PR.

**Comment by majetideepak** [comment]
@amitkdutta Can you merge this PR? Thanks.

**Comment by Yuhta** [comment]
I don't have the merger permission in this repo

**Comment by isadikov** [comment]
Would it be possible to merge this PR? It would unblock another one that I am working on. Thanks.

**Comment by majetideepak** [comment]
@mbasmanova can you please help merge this?

**Comment by aditi-pandit** [comment]
> @isadikov @majetideepak What's the impact on CI run time if any? Which tests are we going to add to CI? Do they pass reliably?

@mbasmanova : Right now we need to add this flag to add any PARQUET testing to Prestissimo. As Parquet is a very popular format that needs to be well tested with Prestissimo, this change is required.

**Review by aditi-pandit — APPROVED** [review]
Thanks for this change @isadikov !

**Comment by isadikov** [comment]
Could someone from @prestodb/committers merge this PR? Thanks.

**Review by ajaygeorge — APPROVED** [review]

**Comment by mbasmanova** [comment]
@majetideepak @amitkdutta @Yuhta 

Deepak, it looks like e2e tests may not work when PRESTO_ENABLE_PARQUET is OFF. Is this so? Would it be possible to fix?

**Comment by majetideepak** [comment]
@mbasmanova We need to come up with a way to disable the corresponding Java tests. It is not clear how we can translate a CMake option to the Java side. I will look into this.

**Pull request merged**
